### PR TITLE
Implement parsing of arguments for create event feature

### PR DESCRIPTION
**Pull request by jordankanghm**
Also added some storage files to their respective packages.

Lastly, updated the user guide for create events feature.

### CODE REVIEW

File: src/main/java/seedu/address/model/event/DateTime.java
Comment (brein62) [inline src/main/java/seedu/address/model/event/DateTime.java:100]: see my PR -- the parsing method for DateTime is changed

**Review by brein62 — CHANGES_REQUESTED** [review]
overall refactoring LGTM, but i won't merge yet due to conflicts in the implementation of DateTime

File: src/main/java/seedu/address/model/event/DateTime.java
Comment (brein62) [reply src/main/java/seedu/address/model/event/DateTime.java:100]: we try to reconcile these changes before we merge anything

**Comment by jordankanghm** [comment]
lgtm

**Review by brein62 — APPROVED** [review]
looks good, really good lookin

**Comment by jordankanghm** [comment]
THANK YOU

**Comment by brein62** [comment]
looks good, the most good lookin so far

File: src/main/java/seedu/address/model/event/Description.java
Comment (ruo-x) [inline src/main/java/seedu/address/model/event/Description.java:19]: wowie look so complex

File: src/main/java/seedu/address/model/event/EventName.java
Comment (ruo-x) [inline src/main/java/seedu/address/model/event/EventName.java:16]: can consider changing name to event name?

**Review by ruo-x — APPROVED** [review]
damn tiny suggestion to further improve the clarity of javadoc.

File: src/main/java/seedu/address/model/event/EventName.java
Comment (jordankanghm) [reply src/main/java/seedu/address/model/event/EventName.java:16]: ok i change for u

**Pull request merged**
